pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -70,7 +70,7 @@ type BindMechanism interface {
 }
 
 type podNICImpl struct {
-	driver NetworkHandler
+	handler NetworkHandler
```

Comment:
I think this change should be done in the previous commit. That would address @dankenigsberg 's comment in https://github.com/kubevirt/kubevirt/pull/4977/commits/d7fbf2cba4ce4ede1f1ccdcd4df30eb726c4e40d#r580912130 .

Reply:
you've beat me to it!

Reply:
Done, removed driver from whole PR.